data/replaced_tags.txt

```diff
@@ -24,3 +37,21 @@ diet:vegan=only : cuisine=vegan
 diet=vegan : cuisine=vegan
 
 aerodrome:type=international : aerodrome=international
+
+amenity=parking_entrance : amenity=parking
+amenity=parking_space : amenity=parking
+amenity=garages : building=garage
+amenity=garage : building=garage
+building=garages : building=garage
+
+historic=cannon : historic=memorial
+historic=technical_monument : historic=memorial
+historic=shield : historic=memorial
+historic=war_memorial : historic=memorial
+
+leisure=resort : tourism=resort
```

Comment:
# OSM recommends leisure=resort instead of tourism=resort
# but we already have the type for tourism and don't have one for leisure.

Reply:
done